internal/pkg/deploy/pipeline.go

```diff
@@ -10,11 +10,18 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+
 	"github.com/aws/aws-sdk-go/aws/arn"
 )
 
 const (
 	fmtInvalidRepo = "unable to locate the repository URL from the properties: %+v"
+	// Redefining these consts from the cli package here so as to avoid an import cycle
```

Comment:
huh I wonder if all of these constants should just be defined in the `manifest` pkg since it's the common consumed package between `deploy` and `cli`?

Reply:
good idea! done.